amundsen_application/static/webpack.common.ts

```diff
@@ -1,10 +1,29 @@
 import * as path from 'path';
+import * as fs from 'fs';
 import * as webpack from 'webpack';
 import HtmlWebpackPlugin from 'html-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 
 import appConfig from './js/config/config';
 
+const walkSync = (dir, filelist = []) => {
+  fs.readdirSync(dir).forEach(file => {
+    filelist = fs.statSync(path.join(dir, file)).isDirectory()
+      ? walkSync(path.join(dir, file), filelist)
+      : filelist.concat(path.join(dir, file));
+  });
+  return filelist;
+};
+const templatesList = walkSync('templates');
+const htmlWebpackPluginConfig = templatesList.map(file => {
+  return new HtmlWebpackPlugin({
+      filename: file,
+      template: file,
+      config: appConfig,
+      inject: false,
+    });
+});
```

Comment:
Because HtmlWebpackPlugin can't process entire directories we have to process them all individually. `walkSync` will return all files in a directory and then create an `HtmlWebpackPlugin` config for each file.

Reply:
Can you clarify what purpose this serves in regards to the fix that needs to made?

Reply:
By default you would need to add one block per template. We currently have four.

```
new HtmlWebpackPlugin({	      ...htmlWebpackPluginConfig,
        template: path.join(__dirname, 'templates/index.html'),	
        filename: 'templates/index.html',	
        config: appConfig,	
        inject: false,	
      }),
```